---
title: "Next-Generation Sequencing of Influenza Viruses in a Household Cohort Accurately Identifies Transmission Pairs and Reveals a Bottleneck Size of Close to One"
authors:
- Ryan-Malosh
date: "2016-10-24T00:00:00Z"
doi: "https://doi.org/10.1093/ofid/ofw194.121"

# Schedule page publish date (NOT publication's date).
publishDate: "2020-08-18T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: Open Forum Infectious Diseases
publication_short: Open Forum Infect Dis

abstract: "Background. A detailed understanding of influenza virus transmission is central to the evaluation of public health interventions. Household transmission is a major driver of influenza epidemics. However, individuals remain at risk for community-acquired infection even while they are exposed to an infected household contact. We hypothesized that sequence data could be used to differentiate community and household-acquired viruses, improving the specificity of transmission studies.


Methods. We used the Illumina platform to sequence 192 influenza A positive throat and nasal swabs collected within 7 days of illness onset over 4 influenza seasons in the HIVE study, a prospective cohort of households with ≥2 children that each year includes 250–350 families. We have developed and robustly benchmarked a variant calling pipeline that can identify intrahost single nucleotide variants (iSNV) as a rare as 0.5% in patient-derived influenza populations with >99.95% specificity.


Results. Intrahost viral diversity was quite low. The average number of iSNV per sample at >0.5% frequency was 10 (range 1–135, IQR 4–9) and did not differ appreciably with day of sampling or vaccination status. Pearson correlation of shared iSNV accurately distinguished 23 H1N1 and 34 H3N2 household transmission pairs from community samples. The pairwise genetic distances of influenza populations from these transmission pairs were sufficiently low relative to a null distribution of distances generated each season from the entire cohort that we could identify transmission pairs with >95% confidence. Across all transmission pairs, 424 iSNV were polymorphic, and 30.2% of iSNV were found in both members of a pair. Maximum likelihood optimization and a simple binomial model estimated that a bottleneck size of <10 transmitted viruses is most consistent with the number and frequency of shared variants in our cohort.


Conclusion. We have used deep sequencing to identify transmission pairs in a household-based cohort. Our data suggest that the size of the transmission bottleneck is much lower than previously thought and that infections originating from more than one virus are relatively uncommon in this population. This study is the most comprehensive to date of the genetics and molecular epidemiology of influenza virus transmission."

# Summary. An optional shortened abstract.
# summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Orthomyxoviridae
- Massively-parallel genome sequencing

featured: false

links:
- name: Online Access
  url: https://academic.oup.com/ofid/article/3/suppl_1/1741/2637597
# url_pdf: 
# url_code: '#'
# url_dataset: '#'
# url_poster: '#'
# url_project: ''
# url_slides: ''
# url_source: '#'
# url_video: '#'

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# image:
#   caption: ''
#   focal_point: ""
#   preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
# projects: 

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---